xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/FormContainerElement.java

```diff
@@ -71,9 +71,9 @@ protected WebElement getFormElement()
         return this.formElement;
     }
 
-    public void fillFieldsByName(Map<String, String> valuesByNames)
+    public void fillFieldsByName(LinkedHashMap<String, String> valuesByNames)
     {
-        Map<WebElement, String> valuesByElements = new LinkedHashMap<>((int) (valuesByNames.size() / 0.75));
+        LinkedHashMap<WebElement, String> valuesByElements = new LinkedHashMap<>((int) (valuesByNames.size() / 0.75));
```

Comment:
I'd go for removing this `valuesByNames.size() / 0.75` stuff, we don't need any kind of optimization here...

Reply:
Done in https://github.com/xwiki/xwiki-platform/pull/3675/commits/22aebacc8cf2b8325c616b122595ffaef5c85bae :+1: 
I checked out online what this optimization is about because it's not easy to understand without comments or prior knowledge ^^'